recipe_box/src/main/java/com/spr/recipe_box/Service/Services/CatService.java

```diff
@@ -0,0 +1,35 @@
+package com.spr.recipe_box.Service.Services;
+
+import com.spr.recipe_box.Constant.Constants;
+import com.spr.recipe_box.Helper.StreamBytesHelper;
+import com.spr.recipe_box.Service.Interface.ICat;
+import org.springframework.http.*;
+import org.springframework.stereotype.Component;
+import java.io.*;
+import java.net.URL;
+
+@Component
+public class CatService implements ICat {
+    StreamBytesHelper streamBytesHelper = new StreamBytesHelper();
+
+    public HttpEntity<byte[]> RandomCatPicture() throws IOException {
+        URL url = new URL(Constants.CAT_URL + Constants.CAT);
+        byte[] response = streamBytesHelper.getBytes(url);
+        HttpHeaders headers = getHttpHeaders(response);
+        return new HttpEntity<byte[]>(response, headers);
+    }
+
+    public HttpEntity<byte[]> RandomCatGif() throws IOException {
+        URL url = new URL(Constants.CAT_URL + Constants.GIF);
+        byte[] response = streamBytesHelper.getBytes(url);
+        HttpHeaders headers = getHttpHeaders(response);
+        return new HttpEntity<byte[]>(response, headers);
+    }
+
+    private HttpHeaders getHttpHeaders(byte[] response) {
+        HttpHeaders headers = new HttpHeaders();
```

Comment:
it looks like this is used in multiple files, I'm not sure but you might want to put it in a helper class or something so that you could call it from there instead of rewriting it.